frontend/javascript/basket.js

```diff
@@ -0,0 +1,121 @@
+import { createTemplate } from './products.js';
+
+let cartList = [];
+
+function removeCartItem(templateClone2, currentProductId) {
+    let cartRemove_btn = templateClone2[templateClone2.length - 1].querySelector('.cart-remove');
+
+    cartRemove_btn.addEventListener('click', function () {
+        sessionStorage.removeItem(currentProductId);
+        cartRemove_btn.parentElement.remove();
+        updateTotal();
+    });
+}
+
+function updateTotal() {
+    let cartBoxes = document.querySelector('.cart-content').querySelectorAll('.cart-box');
+
+    let totalPrice = document.querySelector('.total__price');
+    let total = 0;
+
+    cartBoxes.forEach((cartBox) => {
+        let priceElement = cartBox.querySelector('.cart-box__product_price');
+        let price = parseFloat(priceElement.innerHTML.replace('£', ''));
```

Comment:
query: shouldn't these calculations all be done based on what's in the session storage, rather than by fetching text out of the markup?  wouldn't that be simper?